### DISCUSSION TITLE
Better understanding the json output

### DISCUSSION THREAD

**Discussion by galleon** [mindee/doctr#923] [Q&A]
First thanks for making `docTR` available ! I discovered it yesterday by magic ... so I do apologise in advance for dummy questions.

I have noticed that your json structure in `basically` page - blocks - lines - words

  - words do have a value :-), a confidence and a geometry. All good !
  - lines seem to be a container for words and have a geometry as well

I  am wondering (this is the question) how blocks are constructed ... It seems to be a group of lines by column ? Any feedback on how there are built. Would there be a way to reconstruct the pseudo table in my document (attached) ? My solution so far would be to process each block until I detect a country (using spacy)

[license.pdf](https://github.com/mindee/doctr/files/8717150/license.pdf)

Cheers
Guillaume

**Comment by frgfm** [comment]
Hello @galleon 👋 

So blocks are the equivalent of paragraphs, they are sets of lines 👍 
However I don't understand your question about the pseudo table? You mention a solution but a solution to what problem? 😅 What are you trying to do?

Cheers ✌️

**Comment by galleon** [comment]
Hello @frgfm 

in my attached file, I do consider that the 2-columns list of addresses can be seen as a table (with no solid line around each cell) so I was wondering if you have a way to address tables in general (and this kind of « table » in particular) in doctr?

**Reply by frgfm** [reply]
Oh I see your question now 😅 
So at the current state, there isn't a table comprehension feature. But this is something we've been actively thinking about over the past months! But that won't be a short term feature addition, there is a lot to do before then :)
